src/components/sbb-train/sbb-train.custom.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface InterfaceSbbTrainAttributes {
+  direction: 'LEFT' | 'RIGHT';
```

Comment:
Please use lower case
```suggestion
  direction: 'left' | 'right';
```